### ISSUE TITLE
Mito closed Streamlit Expanders on change

### ISSUE THREAD

**Issue by tylerjrichards** [mito-ds/mito#891]
**Describe the bug**
tldr is that when Mito is used inside a Streamlit expander in a Streamlit app, on any change to Mito the expander closes. I'd like to include Mito in my app to optionally let users play around with the data, but can only get it to work inside the main Streamlit bits, not inside expanders

**To Reproduce**

with st.expander("check this out"):
     df, code = spreadsheet(orig_df)
     
     

Please include the relevant dataset if the bug you encountered is dataset specific. Make sure to anonymize the data properly.

**Expected behavior**
I'd expect the expander to stay open while I play around with Mito

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 11]  Mac ofc 
 - Browser [e.g. Chrome, Firefox] Chrome
 - Mito Version [e.g. 0.3.331] (you can find this with `pip list`) 0.1.504

**Additional context**

**Comment by naterush** [comment]
This has been fixed!
